src/bug-filing/components/bug-filing-choice-group.tsx

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { ChoiceGroup, IChoiceGroupOption } from 'office-ui-fabric-react/lib/ChoiceGroup';
+import * as React from 'react';
+
+import { UserConfigMessageCreator } from '../../common/message-creators/user-config-message-creator';
+import { NamedSFC } from '../../common/react/named-sfc';
+import { UserConfigurationStoreData } from '../../common/types/store-data/user-configuration-store';
+import { BugFilingService } from '../types/bug-filing-service';
+
+export type BugFilingChoiceGroupProps = {
+    deps: BugFilingChoiceGroupDeps;
+    userConfigurationStoreData: UserConfigurationStoreData;
+    bugFilingServices: BugFilingService[];
+};
+
+export type BugFilingChoiceGroupDeps = {
+    userConfigMessageCreator: UserConfigMessageCreator;
+};
+
+export const BugFilingChoiceGroup = NamedSFC<BugFilingChoiceGroupProps>('BugFilingChoiceGroup', props => {
+    const { userConfigMessageCreator } = props.deps;
+
+    const getOptions: () => IChoiceGroupOption[] = () => {
+        return props.bugFilingServices.map(service => {
+            return {
+                key: service.key,
+                text: service.displayName,
+                ariaLabel: service.displayName,
```

Comment:
Is an `ariaLabel` necessary here? I thought it was preferable to omit it if a proper semantic `<label for="id">` with appropriate text existed, which office fabric is intending to provide as part of their `ChoiceGroup` implementation.

Reply:
Done.